stdlib/2and3/rlcompleter.pyi

```diff
@@ -0,0 +1,6 @@
+# Stubs for rlcompleter
+
+from typing import Optional
+
+class Completer:
+    def complete(self, text: str, state: int) -> Optional[str]: ...
```

Comment:
Actually `type(rlcompleter.Completer().complete(u'rl', 0)) == str`, which is weird, but in the stl. `text` can be unicode btw, added in 23d4fe8.